__tests__/implementation.test.js

```diff
@@ -292,6 +292,362 @@ test('batch endpoint (multiple requests)', async () => {
     });
 });
 
+test('batch endpoint (multiple requests) with secondaryBatchKey', async () => {
+    const config = {
+        resources: {
+            foo: {
+                isBatchResource: true,
+                docsLink: 'example.com/docs/bar',
+                batchKey: 'foo_ids',
+                newKey: 'foo_id',
+                secondaryBatchKey: 'properties',
+                secondaryNewKey: 'property',
```

Comment:
as discussed in https://github.com/Yelp/dataloader-codegen/pull/261#discussion_r632734295, I think the behaviour of this is sufficiently specific enough to warrant a special naming:

> I would then also rename `secondaryKey` -> `propertyMergeKey` since this is getting pretty behaviour specific

Or even just a `mergeProperties` which can be a bool flag, and we just enforce that the endpoint meets our specific contract. Otherwise supporting all different merging strategies is going to make the codegen even more unmanageable than it already is

thoughts?

Reply:
I think `propertyMergeKey` is better than `mergeProperties` bool flag. The endpoint in the original feature request https://github.com/Yelp/dataloader-codegen/issues/75 have `features` argument instead of `properties`. I will change to `propertyMergeKey` then.

Reply:
Or we use `propertyBatchKey` and `propertyNewKey`, feel like that's closer to the original `batchKey` and `newKey`.